lib/build-view.js

```diff
@@ -95,6 +99,14 @@ module.exports = (function() {
     this.detach(true);
   };
 
+  BuildView.prototype.toggle = function(event, element) {
+    if (this.isAttached()) {
+      this.detach(true);
+    } else {
+      this.attach();
```

Comment:
I guess that one looks better, yeah. Though I've mostly seen the ternary operator used when something is actually being returned (i.e. as part of an expression).